perception/image_projection_based_fusion/src/utils/utils.cpp

```diff
@@ -13,8 +13,33 @@
 // limitations under the License.
 
 #include "image_projection_based_fusion/utils/utils.hpp"
+
 namespace image_projection_based_fusion
 {
+Eigen::Vector2d calcRawImageProjectedPoint(
+  const image_geometry::PinholeCameraModel & pinhole_camera_model, const cv::Point3d & point3d)
+{
+  cv::Point2d rectified_image_point = pinhole_camera_model.project3dToPixel(point3d);
+
+  cv::Point2d raw_image_point = rectified_image_point;
+
+  try {
+    bool need_unrectify = false;
+    for (const auto & distortion : pinhole_camera_model.cameraInfo().d) {
+      if (distortion != 0.0) {
+        need_unrectify = true;
+      }
+    }
```

Comment:
```suggestion
  const bool need_unrectify = std::any_of(pinhole_camera_model.cameraInfo().d.begin(),
    pinhole_camera_model.cameraInfo().d.end(),
    [](double distortion) { return distortion != 0.0; });
```
How about this? It would be easier to read, and also will enable making `need_rectify` a const variable.

Reply:
[a86ed7b](https://github.com/autowarefoundation/autoware.universe/pull/7030/commits/a86ed7b1088b6bdd884daa21e519afc6ef0c9c0e)